recipe_scrapers/rutgerbakt.py

```diff
@@ -0,0 +1,88 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+
+
+class RutgerBakt(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "rutgerbakt.nl"
+
+    def author(self):
+        return "Rutger van den Broek"
+
+    def title(self):
+        return self.schema.title().replace(" – recept", "").replace(" – Recept","")
+
+    def category(self):
+        category = self.url.split("rutgerbakt.nl/")[-1].split("/")[0].replace("-recepten", "").replace("-", " ")
+        return category
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        # The yields are all over the place. There is no way to parse this.
+        return 1
```

Comment:
Opinions probably vary, but I'd recommend `return None` here, instead -- to indicate that the scraper implements logic to say that it will not provide reliable `yields` field values.  The comment is a good idea.

Reply:
good point, wasn't sure if None is a valid type, since it behaves differently than a integer or something.